apps/glusterfs/brick_entry.go

```diff
@@ -274,6 +274,12 @@ func addVolumeIdInBrickEntry(tx *bolt.Tx) error {
 		}
 		for _, brick := range volumeEntry.Bricks {
 			brickEntry, err := NewBrickEntryFromId(tx, brick)
+			if err == ErrNotFound {
+				logger.Warning("Volume [%v] links to "+
+					"inexistent brick [%v]. Ignoring.",
```

Comment:
Ok, I won't block on the brackets.

However, inexistant is not a normal english word (see https://en.wiktionary.org/wiki/inexistant - which considers it obsolete usage). Do you mind changing it to nonexistent?